chia/util/streamable.py

```diff
@@ -303,7 +303,9 @@ def recurse_jsonify(
     elif isinstance(d, dict):
         new_dict = {}
         for name, val in d.items():
-            new_dict[name] = next_recursion_step(val, None, **next_recursion_env)
+            new_dict[next_recursion_step(name, None, **next_recursion_env)] = next_recursion_step(
```

Comment:
we need to recurse on the key?  i thought json could only have string keys in objects.

Reply:
Well actually the python `json` driver can handle `int`s as keys I learned (it automatically converts them to strings) but yeah I mean that's what this recursion is accomplishing in the end.  So if we have `dict[bytes32, ...]` then the `next_recursion_step` call will make sure it ends up as hex.  Yeah it'll attempt some silly things if you tried to put a dataclass there for example but the python `dict` implementation or the `json` implementation will catch it.